assets/js/morphdom_callbacks.js

```diff
@@ -5,6 +5,10 @@ const callbacks = {
     for (const attr of from.attributes) {
       if (attr.name.startsWith("data-js-")) {
         to.setAttribute(attr.name, attr.value);
+
+        if (attr.name.startsWith("data-js-set-")) {
+          to.setAttribute(attr.name.replace("data-js-set-", ""), attr.value);
```

Comment:
@jonatanklosko we use the value here?

Reply:
Or do you mean I could read the value from the old attribute and keep it as such?

Reply:
Ah, yeah, exactly! Something like this should do:

```js
const keep = attr.name.replace("data-js-set-", "");
to.setAttribute(keep, from.getAttribute(keep));
```